analyzer/codechecker_analyzer/analyzers/cppcheck/analyzer.py

```diff
@@ -225,6 +226,8 @@ def construct_analyzer_cmd(self, result_handler):
 
             analyzer_cmd.append('--plist-output=' + str(output_dir))
 
+            analyzer_cmd.extend(config.analyzer_extra_arguments)
```

Comment:
Looks like this line was already present a few lines up.

Reply:
Good point, thanks for the comment!